src/main/java/hudson/plugins/sshslaves/RemoteLauncher.java

```diff
@@ -140,4 +141,9 @@ private String makeCommandLine(List<String> cmd, FilePath _workDir) {
     public void kill(Map<String, String> modelEnvVars) throws IOException, InterruptedException {
         // no way to do this
     }
+
+  @Override
+  public boolean isUnix() {
+    return !Functions.isWindows();
```

Comment:
does this actually run on the agent?
if not this returns the same s of the master which is substituting one big for a regression

Reply:
I've remembered why it was not merged.